source/manifest.json

```diff
@@ -17,23 +18,28 @@
 	},
 	"permissions": [
 		"storage",
+		"scripting",
 		"contextMenus",
 		"activeTab",
-		"alarms",
+		"alarms"
+	],
+	"host_permissions": [
 		"https://github.com/*",
 		"https://api.github.com/*"
 	],
-	"browser_action": {
+	"action": {
 		"default_icon": "assets/icon.png"
 	},
-	"optional_permissions": [
+	"optional_host_permissions": [
```

Comment:
Firefox _still_ doesn't support this. The good news is that it should be out on July 9th. I'll wait until then to merge this PR.

https://bugzilla.mozilla.org/show_bug.cgi?id=1766026